docs/user/contributing.rst

```diff
@@ -180,7 +180,7 @@ For PRs that may affect GeoDataset sampling speed, you can test the performance
 This code will download a small (1 GB) dataset consisting of a single Landsat 9 scene and CDL file. It will then profile the speed at which various samplers work for both raw data (original downloaded files) and preprocessed data (same CRS, res, TAP, COG). The important output to look out for is the total time taken by ``train_dataloader_next`` (RandomGeoSampler) and ``val_next`` (GridGeoSampler). With this, you can create a table on your PR like:
 
 ======  ============  ==========  =====================  ===================
-        raw (random)  raw (grid)  preprocessed (random)  preprocessed (grid)
+ state  raw (random)  raw (grid)  preprocessed (random)  preprocessed (grid)
```

Comment:
Could also use "PR", "I/O", or another name here. Just can't be empty apparently.